packages/airnode-node/src/evm/requests/blocking.ts

```diff
@@ -1,80 +1,5 @@
-import flow from 'lodash/flow';
-import keyBy from 'lodash/keyBy';
-import { logger, PendingLog } from '@api3/airnode-utilities';
-import { MAXIMUM_SPONSOR_WALLET_REQUESTS } from '../../constants';
-import { ApiCall, Request, GroupedRequests, LogsData } from '../../types';
+import { GroupedRequests, LogsData } from '../../types';
 
-interface ApiCallsWithLogs {
-  readonly apiCalls: Request<ApiCall>[];
-  readonly logs: PendingLog[];
-}
-
-export function blockRequestsWithWithdrawals([
```

Comment:
Why was `blockRequestsWithWithdrawals` (and the related tests) deleted? We only want to remove the limit on the number of sponsor requests.